### DISCUSSION TITLE
Setting up 'Kaiaulu' package to run unit tests upon installation

### DISCUSSION THREAD

**Discussion by mjgaughan** [sailuh/kaiaulu#236] [Q&A] [answered]
Hi all! Thank you for your work on this tool, I am excited to get started using it. 

I am following the 'Installation' instructions on the README and am running into an issue on Step 3, the unit tests. With 'tests/testthat.R' open, I am able to load the 'testthat' library but am unable to load/install the 'kaiaulu' library, as the processes seem to require. As such, I am unable to run the unit tests to ensure that my repo clone is successful. I am assuming I have made a misstep somewhere in the installation process? Apologies, I am rather new at using R for more intensive computing tasks. Any help with this would be greatly appreciated! Thanks!

**Comment by mjgaughan** [comment, marked as answer]
Apologies, I believe I've found the series of steps necessary to install/set up. Closing discussion!

**Reply by carlosparadis** [reply]
No worries Matt! Please feel free to post questions on Discussions if something doesn't seem right. 👍 That in itself will help me focus on what needs more documentation, or maybe you may have just spotted a bug which is equally appreciated. :)
